server/src/main/java/io/spine/server/event/EventBus.java

```diff
@@ -144,6 +144,15 @@ boolean hasDispatchers(EventClass eventClass) {
         return !dispatchers.isEmpty();
     }
 
+    /**
+     * Obtains a view {@code Set} of events that are known to this {@code EventBus}.
+     *
+     * <p>This set is changed when event dispatchers or handlers are registered or un-registered.
+     */
+    public Set<EventClass> registeredEventClasses() {
```

Comment:
Is this a part of an internal or public API? Looks very internal to me, similar to `hasDispatchers(..)` above.